### ISSUE TITLE
python3 support

### ISSUE THREAD

**Issue by mbukatov** [loadtheaccumulator/glusto#5]
So far glusto works with python 2 only. Is there a plan to either switch or support python 3 as well?

**Comment by loadtheaccumulator** [comment]
Yes. The plan is to support Python3 once some additional changes are in, print statements are swapped for logging calls, etc. I'll leave this issue open and ping you when to that next step.

**Comment by loadtheaccumulator** [comment]
In progress. Targeting March 1. Will make an alpha/beta available before then for testing.
